Branch-SDK-TestBed/build.gradle

```diff
@@ -3,7 +3,7 @@ apply plugin: 'com.android.application'
 dependencies {
     implementation project(':Branch-SDK')
     //debugImplementation 'com.squareup.leakcanary:leakcanary-android:1.+'
-    implementation ('com.google.android.gms:play-services-ads:10.0.1')
+    implementation ('com.google.android.gms:play-services-ads:16.0.0')
```

Comment:
It seems like that version had some issues, is this something you are concerned with?
"Known Issue with October 2, 2018 release
Some of the libraries in the October 2, 2018 release contained an issue in their POM dependencies. This most likely manifests as dex merge issues when some dependency version is updated (even when using the strict-version-matcher-plugin). To resolve, update to the Recommended Minimum Version as follows:"

Reply:
Also, is there a good way to prevent these dependencies from getting a year and a half out of date?

Reply:
Note that this was a dependency in the sample app, not the SDK.  The sample wouldn't compile without making the change.

Reply:
Updating to 17.x.x causes a crash in the Sample app.  I think 16.x.x is as high as we can go here.